pkg/grpc/client.go

```diff
@@ -117,11 +119,6 @@ func NewGRPCClientConn(opts *Options) (*grpc.ClientConn, error) {
 		dialOptions = append(dialOptions, grpc.WithTransportCredentials(cert))
 	}
 
-	if opts.ClientDNSRoundRobin {
```

Comment:
what is the relationship with `grpc_client_dns_roundrobin` config option now? 
probably that config option need be deprecated now as it has no effect?

Reply:
I tried to use `dns:///` as before. But it doesn't work with localhost for some reason. I guess to maintain backwards compatibility we should preserve the existing behavior?

Reply:
but before we couldn't really specify multiple endpoints, right? thus I think we just need mark that option deprecated in docs and stop copying it from `config.Options` to `grpc.Options` ?